appjs/views/mixins/form.js

```diff
@@ -73,21 +73,31 @@ module.exports = {
 
         return inst.save();
       }).then(function() {
-        logger.debug('form finished saving');
-
         if ( action === 'new' ) {
           app.view.success('New '+model_class+' saved');
         } else {
-          app.view.success(model_class+' updates saved');
+          if(!app.view.findNotification('Publishing...')){
+            if(model_class === 'Project'){
+              if(view.model.hasUnpublishedUpdates()){
+                app.view.info(model_class+' updates saved but not published');
+              } else {
+                app.view.success(model_class+' updates saved');
+              }
+            } else {
+              app.view.success(model_class+' updates saved');
+            }
+          }
         }
 
         return view.hook('afterSubmit');
       }).then(function() {
         logger.debug('next: '+next);
         if ( next === 'show' && action === 'new' ) {
-          var updatedFormData = view.alpaca.getValue();
-          delete updatedFormData['slug'];
-          view.formDataOnLoad = updatedFormData;
+          if(model_class === 'Project'){
+            var updatedFormData = view.alpaca.getValue();
+            delete updatedFormData['slug'];
+            view.formDataOnLoad = updatedFormData;
+          }
```

Comment:
This fixes an error that appears when adding new blueprints that was created when the save notification was added. The app was looking for `view.alpaca.getValue()` when the blueprint was saved, but since alpaca is not used there, it was coming back as undefined.